### ISSUE TITLE
Accessing DOCS page very slow

### ISSUE THREAD

**Issue by yashwanth2804** [jsplumb/jsplumb#1117]

**Comment by sporritt** [comment]
Yes I've noticed it can be slow from time to time, particularly the first time you load something. We upgraded it to Docusaurus a while ago, which seemed great, but as time goes on I'm becoming less enamoured with the SPA architecture. It doesn't seem necessary. We might roll back to Docusaurus 1.x.

One particularly vexing thing I have noticed myself is that if you click some link and it's taking a while to load, and you're looking at the blue bar creep across the top of the page, refreshing the browser at that point loads the page instantaneously!  The client side router has already rewritten the URL but something is grinding away. So perhaps the first thing to do is to try to figure out if the client side routing can be switched off.

**Comment by mxmissile** [comment]
Middle clicking works instantly. Yes it opens in new tabs, but it beats waiting for that blue react line.

**Comment by sporritt** [comment]
I can't replicate this anymore. And I can't find any reference in the Docusaurus documentation to a way to switch off the react router and let the site just work via page loads.

**Comment by sporritt** [comment]
I see various other issues that seem to point to the same problem, for instance:

https://github.com/gatsbyjs/gatsby/issues/7876

the behaviour described here correlates with what happens on the jsplumb docs - the URL changes but the transition takes forever.  unfortunately that issue has no resolution...

**Comment by MS3300studios** [comment]
I have the same problem in my own project. Opening the link in another tab allows for instant loading of the page. However if I just click on the link it can take up to 10 seconds to load the page. In the mean time nothing is happening and it looks like the page is unresponsive. Does anyone have a solution to this?
I've also seen multiple people report this problem since 2020. It still hasn't been resolved?
